lib/rules/aria-empty-table-header.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "aria-empty-table-header",
+  "selector": "th, [role=\"rowheader\"], [role=\"columnheader\"]",
+  "matches": "heading-matches",
```

Comment:
This matches is only for `heading` elements (h1-6) so the rule is failing to match the selector. I don't think you need a matches for this rule.